src/main/java/org/apache/commons/lang3/text/translate/JavaUnicodeEscaper.java

```diff
@@ -108,9 +111,11 @@ public JavaUnicodeEscaper(final int below, final int above, final boolean betwee
      *            a Unicode code point
      * @return the hex string for the given codepoint
      */
+    @SuppressWarnings("override.param.invalid") // the function it is overriding may take any codepoint value
```

Comment:
That seems like a bug, or at least a violation of the principle of least surprise. If the superclass can take any codepoint, then this one ought to take any codepoint, as well.